LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 devonsays
```

Comment:
Can you update this from "devonsays" to "Devin Price"?

Reply:
change made and committed